meson.build

```diff
@@ -103,7 +103,8 @@ if get_option('with_wayland') != 'no'
   wayland_dep = dependency('wayland-client', version : '>= 1.11.0',
 			   required : get_option('with_wayland') == 'yes')
   wayland_scanner_dep = dependency('wayland-scanner',
-                                   required : get_option('with_wayland') == 'yes')
+                                   required : get_option('with_wayland') == 'yes',
+                                   native : true)
```

Comment:
what does "native" mean here?

Reply:
that is going to be used by the build machine, not the host system. It's important for cross-compilation.